### PR TITLE
Refactor/fix date database

### PR DESCRIPTION
**Pull request by OscarSanner**
This will resolve the issues we had with sorting by dates as the dates will exist proberly.

### CODE REVIEW

File: app/src/main/java/com/goayo/debtify/model/Group.java [hunk 1]
```diff
@@ -1,7 +1,10 @@
 package com.goayo.debtify.model;
 
+import androidx.databinding.DataBindingUtil;
```
Comment (yenanw) [inline app/src/main/java/com/goayo/debtify/model/Group.java:3]: this shouldn't be here

**Review by yenanw — APPROVED** [review]
just need to removed that one import from androidx in model then we gucci

SeemsGood

File: app/src/main/java/com/goayo/debtify/model/Group.java (hunk 1 above)
Comment (OscarSanner) [reply app/src/main/java/com/goayo/debtify/model/Group.java]: It's goneified!

File: app/src/main/java/com/goayo/debtify/model/Group.java [hunk 2]
```diff
@@ -102,16 +105,16 @@ public void removeUser(Set<User> removeUsers) {
 
     /**
      * Creates a debtTracker and adds it to the list of debtTrackers.
-     *
-     * @param lender        the user who lends out money
+     *  @param lender        the user who lends out money
```
Comment (yenanw) [inline app/src/main/java/com/goayo/debtify/model/Group.java:108]: this space right here, mr officer

**Review by yenanw — APPROVED** [review]
I assume documentations will be fixed later
KKomrade

File: app/src/main/java/com/goayo/debtify/model/Group.java (hunk 1 above)
Comment (OscarSanner) [reply app/src/main/java/com/goayo/debtify/model/Group.java]: Begone unnecessary imports! Begone! The power of Intellij compels you!! 

*holds "Design Patterns - by Gang of Four" up while starting to levitate

File: app/src/main/java/com/goayo/debtify/model/Group.java (hunk 2 above)
Comment (OscarSanner) [reply app/src/main/java/com/goayo/debtify/model/Group.java]: With the great power beset on me by the all mighty intellij, the order is returned and the space... well let's just say I sent it to a farm upstate.

**Review by Puh00 — APPROVED** [review]
looks good

**Pull request merged**